Doc/reference/executionmodel.rst

```diff
@@ -139,8 +139,9 @@ namespace.  Names are resolved in the top-level namespace by searching the
 global namespace, i.e. the namespace of the module containing the code block,
 and the builtins namespace, the namespace of the module :mod:`builtins`.  The
 global namespace is searched first.  If the names are not found there, the
-builtins namespace is searched.  The :keyword:`!global` statement must precede
-all uses of the listed names.
+builtins namespace is searched next. If the names are also not found in the
+builtins namespace, new variables are created in the global namespace.
+The :keyword:`global` statement must precede all uses of the listed names.
```

Comment:
```suggestion
The :keyword:`!global` statement must precede all uses of the listed names.
```

FWIW, I agree with @quazi-irfan that it would be better to not resolve this reference, since `global` is already linked in this paragraph as well as in the very next line, and per agreed convention that in python/docs-community#52 (that @JelleZijlstra himself proposed :) ) we generally only linked once per paragraph (or equivalent information unit).

Reply:
Removed reference.